src/EnergyPlus/Furnaces.cc

```diff
@@ -2768,8 +2768,6 @@ namespace Furnaces {
             // Get fan data
             FanName = Alphas(7);
 
-            errFlag = false;
-
```

Comment:
[src/EnergyPlus/Furnaces.cc:2793]:(style),[redundantAssignment],Variable 'errFlag' is reassigned a value before the old one has been used.